backend/src/handlers/question.ts

```diff
@@ -35,16 +35,17 @@ const create = async (event: APIGatewayEvent): Promise<ProxyResult> => {
 	const body = JSON.parse(event.body || '{}');
 	const params = event.queryStringParameters;
 
-	if (!params?.folderId) {
-		return responses.badRequest({ message: 'Missing folderId parameter' });
+	if (!params?.courseId) {
```

Comment:
shouldn't this check for other parameters too? like `question` and `question options`